File: 백준/Bronze/2562. 최댓값/최댓값.java
import java.util.*;


public class Main {


    public static void main(String[] args) throws Exception {
        Scanner sc = new Scanner(System.in);
        int[] arr = new int[9];
        int max = 0;
        int idx = 0;

        for (int i = 0; i < 9; i++) {
            arr[i] = sc.nextInt();
        }
        for (int i = 0; i < 9; i++) {
            if (arr[i] > max) {
                max = arr[i];
            }
        }
        for (int i = 0; i < 9; i++) {
            if (arr[i] == max) {
                idx = i + 1;
            }
        }

        System.out.println(max);
        System.out.println(idx);


    }
}

File: 백준/Bronze/2562. 최댓값/최댓값.cc
#include <bits/stdc++.h>
using namespace std;

int n, temp = 0, idx;
int arr[9];

int main() {
    for(int i = 0; i < 9; i++) {
        cin >> n;
        if(temp < n) {
            temp = n;
        }
        arr[i] = n;
    }
    for(int i = 0; i < 9; i++) {
        if(arr[i] == temp) idx = i;
    }
    cout << temp << "\n" << idx + 1 << "\n";

    return 0;
}